java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2171,6 +2171,8 @@ public T zinterstore(
      * To perform a <code>zinterstore</code> operation while specifying aggregation settings, use
      * {@link #zinterstore(String, KeysOrWeightedKeys, Aggregate)}
      *
+     * @apiNote When in cluster mode, <code>destination</code> and all <code>keysOrWeightedKeys</code>
```

Comment:
```suggestion
     * @apiNote When in cluster mode, <code>destination</code> and all keys <code>keysOrWeightedKeys</code>
```

Reply:
why is this command the only one modified in this file? what about the other zinterstore and the rest of the commands?

Reply:
Oh, that's a mistake - we don't add same-slot notice to transaction, because it is routed to a single node only.
We don't add that to docs in transaction on other clients too.
If you want me to add - please tell.

Reply:
I think we should, but whatever you think right